### DISCUSSION TITLE
Inertia JS Integration?

### DISCUSSION THREAD

**Discussion by aadrian** [nuejs/nue#70] [Q&A]
Hi,

Any chance considering https://inertiajs.com/ integration?

Thank you.

**Comment by tipiirai** [comment]
I believe it would make more sense to ask this question from the Inertia team. My personal time goes into developing the ecosystem: https://nuejs.org/tools/

**Reply by aadrian** [reply]
@tipiirai Inertia (re)extends the ecosystem by allowing monolith apps to be fast. Please take a look at the demo https://inertiajs.com/demo-application

**Comment by Axellarator** [comment]
As far as I read this:
```
Inertia empowers you to build a modern, JavaScript-based single-page application without the tiresome complexity.
```
I got the feeling, it is pretty complex, starting with installation and integration. I have to master Laravel, Inertiajs, Svelte or React, Ruby and PHP, Javascript wouldn't hurt, too. And why is this mix of systems faster or better than a pure Javascript solution is beyond me. 
Let's stick with nue development.
